server/controllers/languages/language.js

```diff
@@ -0,0 +1,4 @@
+// eslint-disable-next-line no-unused-vars
+exports.language = (req, res, next) => {
```

Comment:
```suggestion
exports.getAllLanguages = (req, res, next) => {
```

#### I think we dealing to be constant on naming

Reply:
consistent* is the word fatma *^*

Reply:
yeah @Amoodaa

Reply:
done